mfr/server/handlers/export.py

```diff
@@ -37,61 +37,49 @@ class ExportHandler(core.BaseHandler):
             cache_file_path_str = '/export/{}.{}'.format(self.cache_file_id, self.exporter_name)
         else:
             cache_file_path_str = '/export/{}'.format(self.cache_file_id)
-        self.cache_file_path = await self.cache_provider.validate_path(cache_file_path_str)
-
-        self.source_file_path = await self.local_cache_provider.validate_path(
-            '/export/{}'.format(self.source_file_id)
-        )
-
-        self.output_file_id = '{}.{}'.format(self.source_file_path.name, self.format)
-        self.output_file_path = await self.local_cache_provider.validate_path(
```

Comment:
need to make sure all unnecessary path stiff is removed. Only remote cache and osf provider should remain.